src/components/search/SideFacetDropdown.vue

```diff
@@ -326,12 +352,16 @@
       },
 
       paramsForFacets() {
-        return {
+        const params = {
           ...this.$store.state.search.apiParams,
           rows: 0,
           profile: 'facets',
           facet: this.name
         };
+        if (this.search) {
+          params[`f.${this.name}.facet.limit`] = 125000;
```

Comment:
Perhaps also move this number to a variable, like you did with the `MAXIMUM_VALUES_DISPLAY_COUNT`?